scalafmt-core/shared/src/main/scala/org/scalafmt/rewrite/RedundantBraces.scala

```diff
@@ -108,24 +108,29 @@ class RedundantBraces(ftoks: FormatTokens) extends FormatTokensRewrite.Rule {
       val rt = ft.right
       new Token.LeftParen(rt.input, rt.dialect, rt.start)
     } // we will not keep it but will hint to onRight
-    ft.meta.rightOwner match {
-      case t: Term.Function if t.tokens.last.is[Token.RightBrace] =>
-        if (!okToRemoveFunctionInApplyOrInit(t)) null
-        else if (okToReplaceFunctionInSingleArgApply(t)) replace
-        else removeToken
-      case t: Term.Block =>
-        if (okToReplaceBlockInSingleArgApply(t)) replace
-        else if (processBlock(t)) removeToken
-        else null
-      case _: Term.Interpolate =>
-        if (processInterpolation) removeToken else null
-      case meta.Importer(_, List(x))
-          if !(x.is[Importee.Rename] || x.is[Importee.Unimport]) ||
-            style.runner.dialect.allowAsForImportRename &&
-            (ConvertToNewScala3Syntax.enabled ||
-              !x.tokens.exists(_.is[Token.RightArrow])) =>
-        removeToken
-      case _ => null
+    if (ft.left.is[Token.MacroSplice] || ft.left.is[Token.MacroQuote]) {
+      null
+    } else {
+      ft.meta.rightOwner match {
+        case t: Term.Function if t.tokens.last.is[Token.RightBrace] =>
+          if (!okToRemoveFunctionInApplyOrInit(t)) {
+            null
+          } else if (okToReplaceFunctionInSingleArgApply(t)) replace
```

Comment:
what's changed here? others are simple indentation. if nothing, can we restore?

Reply:
Restored